samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/FeatureLayers.razor

```diff
@@ -1,133 +1,54 @@
 ﻿@page "/feature-layers"
+<PageTitle>FeatureLayer</PageTitle>
 
-<PageTitle>Feature Layers</PageTitle>
-<h1>Feature Layers</h1>
+<h1>Events</h1>
```

Comment:
Does this header need adjustment? Is this a more general events pattern, in this case adding graphics, or would it be better to adjust the events header specify to adding graphics?